owner/manager_test.go

```diff
@@ -41,9 +41,9 @@ func TestT(t *testing.T) {
 
 func checkOwner(d DDL, fbVal bool) (isOwner bool) {
 	manager := d.OwnerManager()
-	// The longest to wait for 3 seconds to
+	// The longest to wait for 30 seconds to
```

Comment:
Is the 30s too long? If the total tests time-out is caused, then it is not easy to find the reason for the test time-out.

Reply:
I think it's ok since this package doesn't contain too many tests.